docs/conventions.rst

```diff
@@ -11,7 +11,7 @@ Coding conventions
 
 We follow most of the practices as detailed in the `Mozilla webdev
 bootcamp guide
-<http://mozweb.readthedocs.org/en/latest/coding.html>`_.
+<https://mozweb.readthedocs.io/en/latest/coding.html>`_.
```

Comment:
404. I think the URL should be https://mozweb.readthedocs.io/en/latest/guide/development_process.html